google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

```diff
@@ -493,6 +496,28 @@ public StatementResult statementShowTransactionIsolationLevel() {
     return resultSet("transaction_isolation", "serializable", SHOW_TRANSACTION_ISOLATION_LEVEL);
   }
 
+  @Override
+  public StatementResult statementSetProtoDescriptors(byte[] protoDescriptors) {
+    Preconditions.checkNotNull(protoDescriptors);
+    getConnection().setProtoDescriptors(protoDescriptors);
+    return noResult(SET_PROTO_DESCRIPTORS);
+  }
+
+  @Override
+  public StatementResult statementSetProtoDescriptorsFilePath(String filePath) {
+    Preconditions.checkNotNull(filePath);
+    getConnection().setProtoDescriptorsFilePath(filePath);
+    return noResult(SET_PROTO_DESCRIPTORS_FILE_PATH);
+  }
+
+  @Override
+  public StatementResult statementShowProtoDescriptors() {
```

Comment:
I think that it would be good to also have a `SHOW PROTO_DESCRIPTORS_FILE_PATH` command to be consistent.

Reply:
Sure. I have added a new client side statement `SHOW VARIABLE PROTO_DESCRIPTORS_FILE_PATH`. This statement would return the proto descriptors file path as output.